force-app/main/default/classes/ProgramSettingsController_TEST.cls

```diff
@@ -399,6 +399,46 @@ private with sharing class ProgramSettingsController_TEST {
         }
     }
 
+    /****************************************************************************************
+     * @description Test method to verify deleteAutoEnrollmentMapping() deletes auto
+     * enrollment on affiliation mappings
+     ****************************************************************************************/
+    @isTest
+    private static void deleteAutoEnrollmentMapping() {
+        AutoEnrollmentMappingService.instance = new STUB_AutoEnrollmentMappingService();
+        Test.startTest();
+        String accountRecordTypeDeveloperName = ProgramSettingsController.deleteAutoEnrollmentMappings(
+            'Mapping_Name_1'
+        );
+        Test.stopTest();
+
+        System.assertEquals(
```

Comment:
This check is performed in AutoEnrollmentMappingService_TEST. The delete method only returns a string, not an entire model. This would be a good case for a jest test